trimmer_decor.rb

```diff
@@ -1,6 +1,6 @@
 require_relative 'base_decor'
 class TrimmerDecorator < BaseDecorator
   def correct_name
-    @nameable.correct_name[0..10]
+    @nameable.correct_name.strip[0...11]
```

Comment:
- [ ]  as the previous reviewer asked you method is still returning 11 characters you should fix this by changing from `@nameable.correct_name.strip[0...11]` to `@nameable.correct_name.strip[0...10]`

Reply:
Done